cms/management/commands/create_initial_data.py

```diff
@@ -24,8 +40,20 @@ def collect_initial_data_functions(self):
                         break
         return functions
 
+    def output(self, app_name, verbosity, done=False, result=False):
```

Comment:
Can we make boolean parameters keyword-only so we pass `self.output(..., done=True, result=False)` instead of `self.output(..., True, False)`?